### PR TITLE
Replace non-responsive code and unify index actions

### PR DESCRIPTION
**Pull request by saraycp**
Move all the user profile actions into the `index_actions/` directory and remove the non-responsive code.

-----------------

If this PR requires any particular action or consideration before deployment,
please check the reasons or add your own to the list:

* [ ] Requires a database migration that can cause downtime. Postpone deployment until maintenance window[1].
* [ ] Contains a data migration that can cause temporary inconsistency, so should be run at a specific point of time.
* [ ] Changes some configuration files (e.g. options.yml), so the changes have to be applied manually in the reference server.
* [ ] A new Feature Toggle[2] should be enabled in the reference server.
* [ ] Proper documentation or announcement has to be published upfront since the introduced changes can confuse the users.

[1] https://github.com/openSUSE/open-build-service/wiki/Deployment-of-build.opensuse.org#when-there-are-migrations
[2] https://github.com/openSUSE/open-build-service/wiki/Feature-Toggles-%28Flipper%29#you-want-real-people-to-test-your-feature

### CODE REVIEW

File: src/api/app/views/webui/user/index_actions/_edit_account.html.haml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  %li.nav-item
+    = link_to(account_edit_link, class: 'nav-link', title: 'Edit Your Account') do
+      %i.fas.fa-lg.mr-2.fa-user-edit
+      %span.nav-item-name Edit your account
+- else
+  %li.nav-item
+    = link_to('#', data: { toggle: 'modal', target: '#edit-modal' }, class: 'nav-link', title: 'Edit Your Account') do
+      %i.fas.fa-lg.mr-2.fa-user-edit
+      %span.nav-item-name Edit Your Account
+
```
Comment (dmarcoux) [inline src/api/app/views/webui/user/index_actions/_edit_account.html.haml:17]: This line should be removed.

File: src/api/app/views/webui/user/index_actions/_edit_account.html.haml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  -# Behind a proxy or not, with this Feature Flag `user_profile_redesign` we go to the edit account form.
+  %li.nav-item
+    = link_to(edit_account_user_path(user), class: 'nav-link', remote: true, title: 'Edit Your Account') do
+      %i.fas.fa-lg.mr-2.fa-user-edit
+      %span.nav-item-name Edit Your Account
+- elsif account_edit_link.present?
+  %li.nav-item
+    = link_to(account_edit_link, class: 'nav-link', title: 'Edit Your Account') do
+      %i.fas.fa-lg.mr-2.fa-user-edit
+      %span.nav-item-name Edit your account
```
Comment (dmarcoux) [inline src/api/app/views/webui/user/index_actions/_edit_account.html.haml:11]: To be consistent with the other `Edit Your Account` links.

```suggestion
      %span.nav-item-name Edit Your Account
```

File: src/api/app/views/webui/user/index_actions/_edit_account.html.haml (hunk 1 above)
Comment (saraycp) [reply src/api/app/views/webui/user/index_actions/_edit_account.html.haml:17]: done

File: src/api/app/views/webui/user/index_actions/_edit_account.html.haml (hunk 2 above)
Comment (saraycp) [reply src/api/app/views/webui/user/index_actions/_edit_account.html.haml:11]: done

**Comment by dmarcoux** [comment]
@saraycp Did you push? I don't see the changes.

**Comment by saraycp** [comment]
> @saraycp Did you push? I don't see the changes.

now yes :sweat_smile:

**Review by dmarcoux — APPROVED** [review]

**Pull request merged**